frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/CovidAoEForm.tsx

```diff
@@ -98,9 +98,13 @@ const CovidAoEForm = ({
                   ? SYMPTOM_SUBQUESTION_ERROR
                   : undefined
               }
+              className={showSymptomOnsetDateError ? "margin-left-0" : ""}
```

Comment:
Noticed that there was some extra margin styling getting applied onto the error states so added these overrides here